tests/config-tests.js

```diff
@@ -47,6 +47,7 @@ describe('Config schema tests', () => {
                     'androidNewStateKillSwitch',
                     'windowsDownloadLink',
                     'windowsExternalPreviewReleases',
+                    'windowsFullScreenMode',
```

Comment:
Do we need the 'windows' prefix on this feature? Even if there's no intent on sharing right now surely we might want it eventually?

Reply:
Good idea, I'll change it to be more generic.

Reply:
Thanks!